stable/osg-frontier-squid/osg-frontier-squid/templates/configmap.yaml

```
@@ -0,0 +1,29 @@
+{{ if .Values.ExtraConf.Enabled }}
+apiVersion: apps/v1
+kind: ConfigMap
+metadata:
+  name: {{ template "osg-frontier-squid.fullname" . }}
+  labels:
+    app: {{ template "osg-frontier-squid.name" . }}
+    chart: {{ template "osg-frontier-squid.chart" . }}
+    release: {{ .Release.Name }}
+    instance: {{ .Values.Instance | quote }}
+data:
+  extra-config.s: |+
+awk --file `dirname $0`/customhelps.awk --source '{
+ setoption("acl NET_LOCAL src", {{ .Values.ExtraConf.acl_NET_LOCAL_src }})
+ setoption("workers", {{ .Values.ExtraConf.Workers }})
+ setoption("cache_mem", {{ .Values.ExtraConf.Cache_Mem }})
+ #setoption("max_filedescriptors", "8192")
+ setoptionparameter("cache_dir", 3, "100000")
+ #setoptionparameter("cache_dir", 2, "/squid/cache/squid${process_number}")
+ setoptionparameter("cache_dir", 2, "/var/cache/squid/squid${process_number}")
+ setoption("cpu_affinity_map", {{ .Values.ExtraConf.Cpu_Affinity_Map }})
+ setoptionparameter("logformat awstats", 3, "kid${process_number}")
+ setoption("visible_hostname", {{ .Values.ExtraConf.Visible_Hostname }})
```

```suggestion
 setoption("visible_hostname", "'`uname -n`'/${process_number}")
```

I think we can hardcode this one and remove it from values.